nodes/project-link.js

```diff
@@ -616,7 +621,7 @@ module.exports = function (RED) {
         RED.nodes.createNode(this, n)
         const node = this
         node.project = n.project
-        node.subscriptionIdentifier = n.project === 'all' ? 2 : 1
+        node.subscriptionIdentifier = (n.broadcast && n.project === 'all') ? 2 : 1
```

Comment:
Was this mistakenly included in PR?

Reply:
Not sure - that fix has already been merged - https://github.com/flowforge/flowforge-nr-project-nodes/pull/4/files#diff-5d286b0aedb2c1e47d6b66c1cebd76e30b8cb2ecf4d95d2f65889a71e6f7a1eaR619

Reply:
yeah - i think it slipped in.

Will mark as reolved (should not me an issue - it is exactly the same code)